.gitignore

```diff
@@ -9,6 +9,7 @@
 *~
 .java-version
 .DS_Store
+.bloop
```

Comment:
I've had a bunch of local gitignore updates for a while so wanted to try to finally get them in, don't know if there's any process for adding to it, but there wasn't anything yet for the setup I use which is vscode with metals scala extension.